app/services/applications/accept.rb

```diff
@@ -108,5 +115,31 @@ def validate_funded_place
         errors.add(:application, I18n.t("application.funded_place_required"))
       end
     end
+
+    def schedule
+      @schedule ||= Schedule.where(identifier: schedule_identifier, cohort:).first || fallback_schedule
+    end
+
+    def fallback_schedule
+      course.schedule_for(cohort:)
+    end
+
+    def validate_permitted_schedule_for_course
+      return if errors.any?
+      return if schedule_identifier.blank?
+      return unless schedule
+
+      unless schedule.course_group.courses.include?(course)
+        errors.add(:schedule_identifier, I18n.t(:schedule_invalid_for_course))
+      end
+    end
+
+    def create_application_state!
+      ApplicationState.create!(
+        application:,
+        lead_provider:,
+        state: "active",
```

Comment:
`state` is `"active"` by default if omitted

Reply:
I think its good to be explicit here instead of relying on the default